BangazonAPI/Controllers/PaymentTypeController.cs

```diff
@@ -0,0 +1,275 @@
+                                CustomerId = reader.GetInt32(reader.GetOrdinal("CustomerId")),
+                                IsActive = reader.GetBoolean(reader.GetOrdinal("IsActive"))
+
+                            };
+                        };
+                    };
+
+
+                    reader.Close();
+
+                    return Ok(paymentTypeToDisplay);
+                }
+            }
+        }
+
+        //  POST: Code for creating a paymentType
+        [HttpPost]
+        public async Task<IActionResult> PostPaymentType([FromBody] PaymentType paymentType)
+        {
+            using (SqlConnection conn = Connection)
+            {
+                conn.Open();
+                using (SqlCommand cmd = conn.CreateCommand())
+                {
+                    cmd.CommandText = @"INSERT INTO PaymentType (AcctNumber, [Name], CustomerId, IsActive)
+                                                    OUTPUT INSERTED.Id
+                                                    VALUES (@AcctNumber, @Name, @CustomerId, @IsActive)";
+                    cmd.Parameters.Add(new SqlParameter("@AcctNumber", paymentType.AcctNumber));
+                    cmd.Parameters.Add(new SqlParameter("@Name", paymentType.Name));
+                    cmd.Parameters.Add(new SqlParameter("@CustomerId", paymentType.CustomerId));
+                    cmd.Parameters.Add(new SqlParameter("@IsActive", paymentType.IsActive));
+
+
+
+                    int newId = (int)cmd.ExecuteScalar();
+                    paymentType.Id = newId;
+                    return CreatedAtRoute("PaymentType", new { id = newId }, paymentType);
+                }
+            }
+        }
+
+        // PUT: Code for editing a paymentType
+        [HttpPut("{id}")]
+        public async Task<IActionResult> PutPaymentType([FromRoute] int id, [FromBody] PaymentType paymentType)
+        {
+            try
+            {
+                using (SqlConnection conn = Connection)
+                {
+                    conn.Open();
+                    using (SqlCommand cmd = conn.CreateCommand())
+                    {
+                        cmd.CommandText = @"UPDATE PaymentType
+                                                        SET AcctNumber = @AcctNumber,
+                                                        Name = @Name,
+                                                        CustomerId=@CustomerId,
+                                                        isActive = @IsActive
+                                                        WHERE id = @id";
+
+                        cmd.Parameters.Add(new SqlParameter("@AcctNumber", paymentType.AcctNumber));
+                        cmd.Parameters.Add(new SqlParameter("@Name", paymentType.Name));
+                        cmd.Parameters.Add(new SqlParameter("@CustomerId", paymentType.CustomerId));
+                        cmd.Parameters.Add(new SqlParameter("@IsActive", paymentType.IsActive));
```

Comment:
Same thing here- let's just keep `isActive` as true and only set it to false with a DELETE request

Reply:
done